metadata-models/src/main/pegasus/com/linkedin/dataJob/DataJobInput.pdl

```diff
@@ -0,0 +1,14 @@
+namespace com.linkedin.dataJob
+
+import com.linkedin.common.DatasetUrn
+
+/**
+ * Inputs consumed by the data job
+ */
+record DataJobInput {
```

Comment:
I wonder if there's any real value breaking this out into a single-field record to simply include it back later. If you're thinking about reusability, maybe creating a common aspect across different job types with both inputs & outputs is a better choice?

Reply:
Breaking out is useful in cases where we don't/can't get the input and output in one MCE. This is currently being reused in our internal samza lineage model as well. Common aspect is a but tricky since different types of jobs might have different kind of entities. In that case, the common aspect will need all kinds of entities. Am I thinking this right?

Reply:
My point is that you already have the `AzkabanJobInput` & `AzkabanJobOutput` aspects to capture these separately. Those aspects have nothing but this single field defined in `DataJobInput` & `DataJobOutput`. I really see the point of that.

Reply:
The idea was that `DataJobInput` can serve as the base and each entity specific `Input` can include additional metadata specific to that entity. It allows entity specific data to be added later. I agree that currently `AzkabanJobInput` doesn't have anything else at this point so we could just define the input directly there. I'll make that change

Reply:
Removed DataJobInput. Agree it wasn't adding any value.